src/msg/node.h

```diff
@@ -25,9 +25,13 @@ class CRNode {
     using job_t           = JobRunner::job_t;
     using strand_ptr_t    = std::shared_ptr<JobRunnerStrand>;
 
-    explicit CRNode() : CRNode("noname") {}
+    explicit CRNode() : CRNode("noname", nullptr) {}
 
-    explicit CRNode(std::string name) : name_(std::move(name)) {}
+    explicit CRNode(std::string name) : CRNode(std::move(name), nullptr) {}
+
+    explicit CRNode(std::shared_ptr<JobRunner> runner) : CRNode("noname", std::move(runner)) {}
+
+    explicit CRNode(std::string name, std::shared_ptr<JobRunner> runner);
```

Comment:
Remove explicit

Reply:
Any reason why?

Reply:
explicit is for single-arg ctor, does it has any effect here?